src/api/config/application.rb

```diff
@@ -13,7 +13,7 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     # Enable rails version 6.0 defaults
```

Comment:
We should change this comment to something like:

`Enable rails version X defaults`

Or we update it every time....

Reply:
yes, rails tries to update it with their update tool, but unfortunately we have too many customization on this file.